docs/backends/gcloud.rst

```diff
@@ -136,6 +138,13 @@ Sets Cache-Control HTTP header for the file, more about HTTP caching can be foun
 Subdirectory in which the files will be stored.
 Defaults to the root of the bucket.
 
+``GS_EXPIRES_ON`` (optional: default is ``timedelta(seconds=86400)``)
+
+The time that a generated URL is valid before expiration. The default is 1 day.
+Public files will return a url that does not expire. Files will be signed by
+the default credentials provided to the client.
```

Comment:
From my reading of the source this **MUST** be a `timedelta`, can we note that?

Also, the name, how about `GS_EXPIRES_IN`? The `ON` language is maybe a bit awkward?

Reply:
The underlying library supports a datetime, timedelta, or an integer number of seconds. I will change it to ON vs IN.

Reply:
This is the doc of that param:

```   
:type expiration: :class:`int`, :class:`long`, :class:`datetime.datetime`,
:class:`datetime.timedelta`
:param expiration: When the signed URL should expire.
```
which gets passed to `get_expiration_seconds` which is:
```python
def get_expiration_seconds(expiration):
    """Convert 'expiration' to a number of seconds in the future.

    :type expiration: int, long, datetime.datetime, datetime.timedelta
    :param expiration: When the signed URL should expire.

    :raises TypeError: When expiration is not an integer.

    :rtype: int
    :returns: a timestamp as an absolute number of seconds.
    """
    # If it's a timedelta, add it to `now` in UTC.
    if isinstance(expiration, datetime.timedelta):
        now = NOW().replace(tzinfo=_helpers.UTC)
        expiration = now + expiration

    # If it's a datetime, convert to a timestamp.
    if isinstance(expiration, datetime.datetime):
        micros = _helpers._microseconds_from_datetime(expiration)
        expiration = micros // 10**6

    if not isinstance(expiration, six.integer_types):
        raise TypeError('Expected an integer timestamp, datetime, or '
                        'timedelta. Got %s' % type(expiration))
    return expiration
```
which I think interprets the integer as an epoch time.

Reply:
Good catch!

Reply:
Updated in docs.